lib/slack/index.js

```diff
@@ -60,7 +60,33 @@ module.exports = (robot) => {
 
   router.post('/actions:dialog_submission:create-issue-dialog', create.dialogSubmit);
 
+  router.post(
+    '/actions:message_action:attach-to-issue',
+    middleware.requireGitHubUser,
+    create.attachToIssueFromMessage.openDialog,
+  );
+  router.post(
+    '/actions:dialog_submission:add-comment',
+    middleware.requireGitHubUser,
+    create.attachToIssueFromMessage.createCommentDialogSubmission,
```

Comment:
I'm not sure how many other dialogs we'll use, but it might be nice to standardize the naming of the methods to something django-ish/rails-y/resourceful. That way we can expect every dialog module has the same methods. Something like:

- `create.attachToIssueFromMessage.open`
- `create.attachToIssueFromMessage.submit`